KarhooUISDK/Screens/BookingRequestScreen/BookingRequestMVP.swift

```diff
@@ -10,7 +10,7 @@ import KarhooSDK
 
 protocol BookingRequestPresenter {
 
-    func load(view: BookingRequestView)
+    func load(view: BookingRequestView, karhooUser: Bool)
```

Comment:
The presenter is the source of truth. It can use the same logic as we have in the ViewController to find out if the Auth method is KarhooUser or not :) 
in the presenter you can have 
```
func load(view: BookingRequestView) {
        self.view = view
        switch Karhoo.configuration.authenticationMethod() {
            case .karhooUser:
                self.karhooUser = true
            default:
                self.karhooUser = false
        }
```